packages/core/src/tools/boundedBuffer.spec.ts

```diff
@@ -26,4 +26,17 @@ describe('BoundedBuffer', () => {
     buffered.drain()
     expect(spy.calls.count()).toEqual(5)
   })
+
+  it('store at most 500 callbacks by default', () => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏this test seems to duplicate the previous one with a different limit. I think we slould keep only one of them

Reply:
I had the same feeling, but I wanted to test the default value as it's important for performance matter.
Don't you think we could remove the parameter since there is not place in the code where we use it?

Reply:
I added the parameter because the limit was reallly high and testing it with the limit could have make tests somewhat slower. But now that the limit is smaller, I think it's fine to remove the parameter and only test the default limit.